<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cut</class>
 <widget class="QWidget" name="Cut">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>770</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>770</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>341</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Logistic&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>786</width>
     <height>194</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="5">
     <widget class="QSpinBox" name="extYbox"/>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>extY</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QSpinBox" name="tempSpinBox"/>
    </item>
    <item row="5" column="0">
     <widget class="QComboBox" name="dirBox"/>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QPushButton" name="rewriteButton">
      <property name="text">
       <string>Rewrite</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QSpinBox" name="spinBox"/>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QPushButton" name="prevButton">
      <property name="text">
       <string>Prev</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QPushButton" name="nextButton">
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="cutButton">
      <property name="text">
       <string>Cut</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QSpinBox" name="nsBox"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="zeroButton">
      <property name="text">
       <string>Zero</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QSpinBox" name="eogSpinBox"/>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>wndLength</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QDoubleSpinBox" name="wndLengthSpinBox"/>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="cycleTimeShiftButton">
      <property name="text">
       <string>Cycle timeSh</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Temperature</string>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <widget class="QSpinBox" name="eyesSlicesSpinBox"/>
    </item>
    <item row="5" column="4">
     <widget class="QDoubleSpinBox" name="eogDoubleSpinBox"/>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="cycleWndLengthButton">
      <property name="text">
       <string>Cycle wndL</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QPushButton" name="windowsButton">
      <property name="text">
       <string>Windows</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QCheckBox" name="checkBox_2">
      <property name="text">
       <string>write windows?</string>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QPushButton" name="paintDistrButton">
      <property name="text">
       <string>Paint Distribution</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>NumOfEyesSlices</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>NumOfDispersion</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="cutEyesButton">
      <property name="text">
       <string>Cut Eyes</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QDoubleSpinBox" name="timeShiftSpinBox"/>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>timeShift</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>ns</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="splitButton">
      <property name="text">
       <string>Split</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="picLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>261</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>1050</width>
     <height>460</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1048</width>
      <height>458</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="browseButton">
      <property name="text">
       <string>Bro&amp;wse</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>paint?</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
